sensor/common/sensor/central_sender_impl.go

```diff
@@ -136,6 +136,7 @@ func (s *centralSenderImpl) send(stream central.SensorService_CommunicateClient,
 
 			if err := wrappedStream.Send(msg.MsgFromSensor); err != nil {
 				log.Errorf("unable to send to stream: %s", err)
+				log.Errorf("message that cause the problem: %+v", msg.MsgFromSensor)
```

Comment:
should we combine the 2 lines into one so error is close to message in case for some log processing tools?

Reply:
That would make the log line pretty long, but I also think this may be a better way. Done in [99dd30b](https://github.com/stackrox/stackrox/pull/15326/commits/99dd30bf2687488bebf39a4ec77e7746b884a78a)